admin/server/projects.go

```diff
@@ -1217,6 +1222,12 @@ func (s *Server) GetCloneCredentials(ctx context.Context, req *adminv1.GetCloneC
 		return nil, status.Error(codes.InvalidArgument, err.Error())
 	}
 
+	claims := auth.GetClaims(ctx)
+	if !claims.Superuser(ctx) && !claims.ProjectPermissions(ctx, proj.OrganizationID, proj.ID).ManageProject {
```

Comment:
Now that this endpoint is no longer just for superusers, can you add a `force_superuser_access` flag and check it similar to this: https://github.com/rilldata/rill/pull/7283